.github/dependabot.yml

```diff
@@ -6,5 +6,7 @@ updates:
     - dependency-type: "all"
   schedule:
     interval: daily
+    time: "04:00"
```

Comment:
This may be affected by the overnight shutdown, should we schedule this at the same time as `apply`, for consistency?

Reply:
Can do :+1: 
I forgot that HMCTS uses RDS for UAT! 🤦‍♂️